blog/_posts/btcpay-server-1-5-0.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Introducing BTCPay Server 1.0.5 - Dashboard and payout processors"
+date: 2022-04-28
+author: pavlenex
+category:
+  - "releases"
+tags:
+  - "btcpay-server"
+  - "release"
+  - "payouts"
+  - "dashboard"
+  - "payout processors"
+  - "update"
+coverImage: "/images/BTCPay1-5-0-desk.png"
+---
+
+After a long wait, we're finally out of our coding cave, ready to introduce a new major BTCPay Server version - 1.5.0! 
+
+This release brings in a brand-new dashboard for easier analytics and better data visualization, payout processors and schedule for later for better payout management payments, batching your payments and refunds, several UX/UI improvements, greenfield API additions and a few non-critical bug fixes.
+
+## New dashboard 📊
+
+We know that having a quick glance at your store's performance is essential. Also we understand that as a user, you would like to have an easy way to access and manage recent invoices and refunds. That's why we're introducing a **dashboard**!
+
+![](/images/1-5-0-dashboard-1.png)
+
+![](/images/1-5-0-dashboard-2.png)
+
+Dashboard visually showcases critical store's data, providing you with a way to better understand your store's performance, so that you can make more informed decisions. Furthermore, a dashboard should help with easier management of recent invoices and pending payouts and refunds.
+
+Currently, the dashboard shows data for an on-chain payment method, pending payouts and refunds, recent invoices & transactions and the data for the crowdfunding app.
+
+![](/images/1-5-0-dashboard-3.png)
```

Comment:
Can you made this lightmode Img ?  For consistency.

Reply:
The inconsistency has been added on purpose to showcase both modes.